cypress/integration/basic-functionalities/local-tests/reporter/1_Create_a_report.spec.js

```diff
@@ -1,105 +1,110 @@
-    });
-  });
+  context('Test for checking if new report and delete buttons are present when in edit mode', () => {
+    it('should be able to access new report and delete buttons', () => {
+      cy.get('table > tbody > :first-child() > :last-child() > a:nth-child(2)').click() // We click on the Edit button on the created report
+      cy.get('[data-test-id="button-create-report"]')
+      cy.get('[data-test-id="button-delete"]')
+      cy.get('[data-test-id="button-back"]').click()
+    })
+  })
 
-  context("Test for creating a new report through the edit mode", () => {
-    it("should be able to create a new report through the edit mode of the previous report", () => {
-      cy.get("table > tbody > :first-child() > :last-child() > a:nth-child(2)").click(); // We click on the edit button on the created report
-      cy.get('[data-test-id="button-create-report"]').click(); // We click on New report button
-      cy.get('[data-test-id="input-name"]').type("Another cypress report"); // We enter the name of the report
-      cy.get('[data-test-id="input-handle"]').type("another_cypress_report"); // We enter the handle of the report
-      cy.get('[data-test-id="input-description"]').type("This is another automated description."); // We enter the description
-      cy.get('[data-test-id="button-save"]').click(); // We click on the save button
-      cy.get('.b-toast-success'); // We check if the success toast appears
-      cy.get('[data-test-id="button-back"]').click(); // We click on the Back button to go on the main page
-    });
-  });
-  context("Test for creating a report with a same handle as another", () => {
-    it("shouldn't be able to create another report with the same handle", () => {
-      cy.get('[data-test-id="button-create-report"]').click(); // We click on New report button
-      cy.get('[data-test-id="input-name"]').type("Name"); // We enter the name of the report
-      cy.get('[data-test-id="input-handle"]').type(consts.cypressCreate); // We enter the handle of the report
-      cy.get('[data-test-id="button-save"]').click(); // We click on the Save button
-      cy.get('.b-toast-danger'); // We check if the danger toast appears
-    });
-  });
-});
+  context('Test for creating a new report through the edit mode', () => {
+    it('should be able to create a new report through the edit mode of the previous report', () => {
+      cy.get('table > tbody > :first-child() > :last-child() > a:nth-child(2)').click() // We click on the edit button on the created report
+      cy.get('[data-test-id="button-create-report"]').click()
+      cy.get('[data-test-id="input-name"]').type('Another cypress report')
+      cy.get('[data-test-id="input-handle"]').type('another_cypress_report')
+      cy.get('[data-test-id="input-description"]').type('This is another automated description.')
+      cy.get('[data-test-id="button-save"]').click()
+      cy.get('.b-toast-success') // We check if the success toast appears
```

Comment:
After updating the report, please, check if the fields contains the correct values